### ISSUE TITLE
Dual core

### ISSUE THREAD

**Issue by guescio** [NuclearInstruments/MVMFirmware#3]
As it has already been mentioned, the EP32 (not the new ESP32-S2 SOC) has two cores, so let’s dedicate one exclusively to the handling of the breathing functions. The software is already well setup to have the breathing all in one function (`onTimerCoreTask()`), so all it takes is a call to `xTaskCreatePinnedToCore()`.

**Comment by cericdahl** [comment]
I agree.  Especially if this allows us to decrease TIMERCORE_INTERVAL_MS.  The current 100ms strikes me as  slow for reaction to e.g. coughing (especially with the averaging currently employed on that interlock).
